undefined_syms.txt

```diff
@@ -2418,6 +2418,15 @@ D_0600AE1C = 0x0600AE1C;
 D_0600B644 = 0x0600B644;
 D_0600F248 = 0x0600F248;
 D_06012A58 = 0x06012A58;
+D_06011138 = 0x06011138;
+D_06011938 = 0x06011938;
+D_06012138 = 0x06012138;
+D_06013684 = 0x06013684;
+D_060152EC = 0x060152EC;
+D_06015C70 = 0x06015C70;
+D_060165F0 = 0x060165F0;
+D_06014220 = 0x06014220;
+D_06014A9C = 0x06014A9C;
```

Comment:
Sorry, one last thing. Don't forget to cleanup undefined_syms.txt. Should be able to remove the entire `ovl_En_Hg` section.

Reply:
No worries, good catch 👍